frontend/app/src/components/reusableComponents/TableComponent.tsx

```diff
@@ -7,7 +7,7 @@ interface TableComponentProps<T> {
     dataSource?: T[];
     onChange?: TableProps<T>["onChange"];
     icon?: React.ReactNode;
-    style: string
+    style: string;
```

Comment:
Lets make this optional
```suggestion
    style?: string;
```